docker/Dockerfile

```diff
@@ -14,7 +18,12 @@ RUN apt-get update \
         systemd \
         ca-certificates \
         adduser \
+    # Install Logging Agent.
     && curl -sS https://dl.google.com/cloudagents/install-logging-agent.sh | REPO_SUFFIX="$REPO_SUFFIX" DO_NOT_INSTALL_CATCH_ALL_CONFIG=true bash \
+    # Store versions in the VERSION file.
+    && echo REPO_SUFFIX=$REPO_SUFFIX >> /VERSION \
+    && dpkg -s google-fluentd | grep Version | sed 's/Version: /GOOGLE_FLUENTD_VERSION=/g' >> /VERSION \
```

Comment:
`sed` will know if it made a substitution, so the following avoids the `grep`: `dpkg -s google-fluentd | sed -n 's/^Version: /GOOGLE_FLUENTD_VERSION=/p' >>/VERSION`...